### PR TITLE
Cinema renames

### PR DESCRIPTION
**Pull request by rr-**
#### Checklist

- [x] I have read the [coding conventions](https://github.com/rr-/Tomb1Main/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] I have added a changelog entry about what my pull request accomplishes, or it is an internal change

#### Description

Increment for #521

### CODE REVIEW

File: src/game/game_demo.c [hunk 1]
```diff
@@ -15,11 +14,67 @@
...
+        .deselect = (bool)(*m_DemoPtr & (1 << 21)),
+        .save = (bool)(*m_DemoPtr & (1 << 22)),
+        .load = (bool)(*m_DemoPtr & (1 << 23)),
+    };
+
+    m_DemoPtr++;
+    return true;
+}
+
+void Game_Demo(void)
```
Comment (walkawayy) [inline src/game/game_demo.c:77]: `Game_Demo_Start` to match `Game_Start`? I guess it's not really a big deal.

File: src/game/camera.c [hunk 2]
```diff
@@ -728,6 +765,36 @@ void Camera_OffsetReset(void)
     g_Camera.additional_elevation = 0;
 }
 
+void Camera_UpdateCutscene(void)
+{
+    PHD_VECTOR campos;
+    PHD_VECTOR camtar;
```
Comment (walkawayy) [inline src/game/camera.c]: `cam_pos` and `cam_tar`?

**Review by walkawayy — APPROVED** [review]

File: src/game/game_demo.c (hunk 1 above)
Comment (rr-) [reply src/game/game_demo.c:77]: Unlike `Game_Cutscene_Start` which is similarly sliced, this routine runs the entire loop (similar to `Game_Pause`).

File: src/game/camera.c (hunk 2 above)
Comment (rr-) [reply src/game/camera.c]: I've no idea what they are ^^' Oh you meant the naming. I'll update.

**Pull request merged**